test/test_torch.py

```diff
@@ -1107,6 +1107,21 @@ def test_tensor_factory(self):
         a = torch.tensor([True, True, False, True, True], dtype=torch.bool)
         b = torch.tensor([-1, -1.1, 0, 1, 1.1], dtype=torch.bool)
         self.assertEqual(a, b)
+        c = torch.tensor([-0.1, -1.1, 0, 1, 0.1], dtype=torch.bool)
+        self.assertEqual(a, c)
+        d = torch.tensor((-.3, 0, .3, 1, 3 / 7), dtype=torch.bool)
+        e = torch.tensor((True, False, True, True, True), dtype=torch.bool)
+        self.assertEqual(e, d)
+        f = torch.tensor((-1, 0, -1.1, 1, 1.1), dtype=torch.bool)
+        self.assertEqual(e, f)
+        g = torch.tensor((float('nan'), 0, -9223372036854775807, 9223372036854775807, 18446744073709551615), dtype=torch.bool)
```

Comment:
Any reason for these numbers? An alternative might to be to use the min/max values in torch.finfo and torch.iinfo.

Reply:
Thank you for your suggestion, It's very helpful, I'll use torch.finfo and torch.iinfo in test code.